src/components/FlickityCarousel.js

```diff
@@ -21,7 +21,7 @@ class FlickityCarousel extends Component {
     const { children, changeCaption, ApplyCarousel } = this.props
     if (ApplyCarousel) {
       const captions = r.map(r.view(r.lensPath(["props", "caption"])), children)
-      this.flkty.on("dragEnd", () => {
+      this.flkty.on("change", () => {
```

Comment:
What was the reason this was changed? We chose dragEnd in the end because there was severe lag on the caption when we changed slide. 😄

Reply:
Ahh interesting. The captions weren't changing on the carousel for me so I thought this was a bug. Maybe because I was clicking on the dots at the bottom not 'dragging' the carousel.